packages/@aws-cdk/aws-events/README.md

```diff
@@ -89,6 +89,10 @@ import { Rule, Schedule } from '@aws-cdk/aws-events';
 import { EcsTask } from '@aws-cdk/aws-events-targets';
 ...
 
+// in default, EcsTask runs on dafault fargate platform version managed by ECS.
+// if you want to specify its version, you can set `platformVersion` in EcsTask's props
+// more information for fargate platform versions:
+//   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/platform_versions.html
```

Comment:
maybe this comment really belongs as a footnote to the code. Or you can always show an example that actually sets that property by creating another target. i.e. `const ecsFargateTarget = new EcsTask...`